src/assets/AddTimeCardPeriod.scss

```diff
@@ -0,0 +1,8 @@
+.add-timecard-period-type {
+  border: 1px solid #b1b4b6;
+  padding: 2px 15px;
+}
+
+.add-timecard-period-type-description {
```

Comment:
I was hoping the name would be a bit more generic (& the relevant stylesheet) so that it would be applicable to other areas of the time card. Maybe something along the lines of '_timecard-description_' as the design shows the same colour being used for time period, notes & time line description.

Reply:
Thanks, yeah that's what I was looking for